src/App.jsx

```diff
@@ -14,70 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, setSelected] = useState('Jam');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
The name `selected` doesn't represent the content of the state.